basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
 
-## Opening a terminal application
+So, it is important to be familiar with how the Terminal works. That is what we'll be discussing in this section. We'll do so by introducing a tool you'll often use when working on projects: git.
+
+Don't worry, though. We'll stick to the basics for now. But you can do many awesome things with the terminal alone! More on that in the unit on UNIX, later in the course.
+
+> ### Following along 
+> You'll get the morst out of this section if you follow along with it on your own device. Try out what we're discussing! If you get stuck, or something doesn't quite do what you were expecting, feel free to ask for help!
+
+## Opening a Terminal application
+To do anything with a Terminal, first you need to open a Terminal window. This can be done in many different ways, sometimes depending on your OS and the software you're using.
+
+### Opening a Terminal directly in Visual Studio Code
+Sometimes, you need the Terminal a lot while coding. In that case, it may be best to open a Terminal window directly in Visual Studio Code. 
+
+To do so on a Mac or Linux device, simply navigate to Terminal > New Terminal or hit Ctrl + Shift + ` (should be three keys at the bottom-left of your keyboard).
+
+![](vscode_newterminal_mac.png)
+
+On a Windows machine, you need to make sure to open the WSL terminal instead of the Windows command prompt. Once you have enabled WSL (refer back [here](../workshop/windows) if you've forgotten how), hit Ctrl + J to open the WSL Terminal.
+
+![](vscode_terminal.png)
+
+Voilà! A brand new Terminal window.
+
+### Opening a separate Terminal window - Mac and Linux
+Sometimes, you may want to use a separate window instead. For example, if you have a second monitor and want to make more space for your code. On a Mac or Linux device, you can simply search for 'Terminal' and open the application. A new terminal window will pop up!
+
+![](mac_terminal.png)
+
+### Opening a separate Terminal window - Windows
+Sometimes, you may want to use a separate window instead. For example, if you have a second monitor and want to make more space for your code. On a Windows device, you can open a Terminal window by pressing the Windows Key + R, and then entering 'ubuntu' in the window that appears. After a small wait, a Terminal window should pop up!
+
+> TODO: insert a screenshot when I have access to a Windows machine.
 
 ## Using `cd` to get to the right spot
+When working in a Terminal, you're always working within a certain folder on your device. All the commands you run will run relative to that folder. The selected folder in a Terminal window is called the 'present working directory'. To find out what folder you have selected, simply type the shorthand for that: `pwd`, and press enter.
```

Comment:
Gebruik consistent directory ipv folder ;-) Dit is de UNIX-terminologie maar wel relevanter voor dit onderwerp.

"selected" lijkt me niet heel gebruikelijk, misschien even omgooien om het handiger te kunnen formuleren. "the directory that you cd'd into" kan ook maar dat kun je zeker niet 2x achtereen gebruiken

"within" kan misschien nadruk krijgen

Reply:
Grappig genoeg heeft wikipedia niks met **present** working directory: https://en.wikipedia.org/wiki/Pwd

Ik zou het in die lijn de "working directory" noemen en het commando dus zoals op wikipedia.